server.js

```diff
@@ -6,7 +6,7 @@ var app = express();
 var server = http.createServer(app);
 var io = require('socket.io').listen(server);
 
-var bug = {};
+var bug = {livetext: "Live", loctext: "York University"};
```

Comment:
The reason I set this to something by default is so that someone can't put it live without it being set... maybe that's not too clever! Having looked at other dynamic variables, there are loads of elements which can go on blank. 

I'll unset this but is it worth looking at not allowing the graphic/element to show if the variable = ''?